packages/web-frontend/cypress/config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dashboardReports": {
+    "allowEmptyResponse": false,
+    "snapshotTypes": ["responseBody", "html"],
+    "urlFiles": ["cypress/config/dashboardReportUrls/default.json"],
+    "urls": []
+  },
+  "mapOverlays": {
+    "allowEmptyResponse": false,
+    "snapshotTypes": ["responseBody"],
```

Comment:
We should create a `cypress/config/mapOverlayUrls/default.json` file and use it as the standard url specification for map overlay tests